docs/quickstarts/project-quickstart.md

```diff
@@ -92,20 +92,22 @@ Office.context.document.getSelectedTaskAsync(
 
 ## Try it out
 
-1. Start the local web server by running the following command:
+> [!NOTE]
+> Office Add-ins should use HTTPS, not HTTP, even when you are developing. If you are prompted to install a certificate after you run one of the following commands, accept the prompt to install the certificate that the Yeoman generator provides.
+
+> [!TIP]
+> If you're testing your add-in on Mac, run the following command before proceeding. When you run this command, the local web server will start.
```

Comment:
I noticed that the sideload instructions are for Windows. Is this quick start available for Mac?